doc/mee-spec.md

```diff
@@ -124,6 +124,44 @@ Sets the baud rate of the given UART.  Returns 0 on success, or -1 on
 failure.  Failure to set the baud rate can render the UART unusable
 until a subsequent coll to `mee_uart_set_baud_rate()` returns success.
 
+### SPI Interface
+
+The SPI interface allows users to perform transfers over the SPI protocol.
+
+`int mee_spi_init(struct mee_spi *spi, int baud_rate)`
+
+Initializes the given SPI. This must be called exactly once before any other
+function in this SPI is called. It is invalid to initialize a SPI more than
+once.
+
+`int mee_spi_transfer(struct mee_spi *spi, struct mee_spi_config *config, int words, char *tx_buf, char *rx_buf)`
```

Comment:
`int words` should be a `size_t`.  Also, calling it `words` is a bit confusing as it's a number of bytes, right?

Reply:
I initially called it "words" because I was initially going to support a variable-length frame size. I'll rename it to `len`.